cnf-certification-test/results/archiver.go

```diff
@@ -0,0 +1,88 @@
+package results
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	tarGzFileNameSuffix = "cnf-test-results.tar.gz"
+)
+
+func generateZipFileName() string {
+	dateTimePrefixFormat := "%02d%02d%02d-%02d%02d%02d"
```

Comment:
Move it to const section.

Reply:
Done. I followed the layout format suggestion from @edcdavid in https://github.com/test-network-function/cnf-certification-test/pull/1200/commits/93040781ec5db491aee76d5bd8ad94f9f144211a